build/edge/kubernetes/README.md

```diff
@@ -0,0 +1,41 @@
+##  Deploy the edge part into a k8s cluster
```

Comment:
Can we create a new file and simply link the file in the README?

Reply:
Yep